lib/state/app_state_container.dart

```diff
@@ -178,6 +178,31 @@ class AppStateContainerState extends State<AppStateContainer> {
     }
   }
 
+  Future<AudioPlayer> playArticleAudio(String audio) async {
```

Comment:
Instead of returning the audioPlayer and thus giving full control to anyone who calls this method, take a callback that can be executed when the audioplayer completes